pkg/webhook/resources/setting/validator.go

```diff
@@ -1814,7 +1814,61 @@ func validateUpgradeConfigHelper(setting *v1beta1.Setting) (*settings.UpgradeCon
 	return config, nil
 }
 
-func validateUpgradeConfigFields(setting *v1beta1.Setting) error {
+func checkNodesDuplicates(nodeNames []string) error {
+	duplicates := ds.SliceFindDuplicates(nodeNames)
```

Comment:
why not just dedupe it with `ds.SliceDedupe`?

Reply:
The returned error message clearly shows what is duplicated, which provides a better user experience.